apps/charterafrica/src/payload/globals/PageConfig.js

```diff
@@ -0,0 +1,259 @@
+const PageConfig = {
```

Comment:
Cool, but I think:

1. We should call this translation strings or something. They're site wide and not page.
2. May be we should include a help text to show where it's being used e.g. showAll -> used to toggle number of visible cards on opportunities page, etc.
3. For these, may be we should also include default values since we don't want the site to break when someone has entered events but hasn't gotten around to entering translations yet.
4. May be we should define a js/json file with all the keys an reuse them in payload as well as data functions. Otherwise spelling mistakes will creep in.